docs/src/generators.md

```diff
@@ -1,49 +1,51 @@
-# Generators
+# Making and Modifying Graphs
 
-## Random Graphs
+*LightGraph.jl* provides a number of methods for creating a graph object,
+including tools for building and modifying graph objects, a wide array of graph
+generator functions, and the ability to read and write graphs from files
+(using [GraphIO.jl](https://github.com/JuliaGraphs/GraphIO.jl)).
 
-*LightGraphs.jl* implements some common random graph generators:
 
-```@autodocs
-Modules = [LightGraphs]
-Pages   = [
-    "generators/randgraphs.jl"
-]
-Private = false
-```
+## Modifying graphs
 
+- `Graph`: Constructor for a simple undirected graph.
+- `DiGraph`: Constructor for a simple directed graph.
+- `add_edge!`: Add an edge to an existing graph.
+- `rem_edge!`: Remove an edge from an existing graph.
+- `add_vertex!`, `add_vertices!`: Add one (or several) vertices to an existing graph.
+- `rem_vertex!`: Remove an existing vertex. Note that for `Graph` and `DiGraph` objects
+this is accomplished via a "swap-and-pop" procedure where the highest-indexed vertex
+is swapped with the vertex to be removed, and only then is the vertex removed. As a
+result, unlike in some libraries (like `iGraph`), vertex indices will not all
+down-shift when a vertex is removed; rather, the only change is that the previously
+highest-index vertex will take on the index of the removed index.
+- `zero`: creates a zero-vertex zero-edge graph of same types as the graph passed
```

Comment:
creates an empty graph, with no vertices and no edges graph with the same types as the input graph.

Reply:
Again, I think we should let the docstrings speak for themselves here.